src/screens/HyperSwitch/Refunds/ShowRefund.res

```diff
@@ -118,17 +119,19 @@ let make = (~id) => {
         overriddingStylesTitle={`text-3xl font-semibold`}
       />}>
       <RefundInfo orderDict={refundData->LogicUtils.getDictFromJsonObject} />
-      <LoadedTable
-        title="Payment"
-        actualData=orderData
-        entity={OrderEntity.orderEntity}
-        resultsPerPage=1
-        showSerialNumber=true
-        totalResults=1
-        offset
-        setOffset
-        currrentFetchCount=1
-      />
+      <UIUtils.RenderIf condition={userPermissionJson.paymentRead !== NoAccess}>
```

Comment:
Are we want completely not to show the table itself

Reply:
Yes Jeeva else the API Call will happen and then the that will be in error state